### PR TITLE
XIVY-6952 clarify how to setup the Linux Designer

### PR DESCRIPTION
**Pull request by ivy-rew**
- document ./install-dependencies.sh
- tell Ubuntu users that we need an X.org session

### CODE REVIEW

**Comment by ivy-rew** [comment]
you can see it in action here: http://192.168.3.52/installation?downloadUrl=https://download.axonivy.com/8.0.1/AxonIvyDesigner8.0.1.96047_Linux_x64.zip&version=8.0.1&product=designer&type=Linux

File: src/app/pages/installation/_linux_designer.twig [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<p>
```
Comment (alexsuter) [inline src/app/pages/installation/_linux_designer.twig:1]: just for your info: this file must be true for all versions below:
- LTS
- LE
- Nightly
- Dev
- Nightly-8
- Sprint
- Nighlty-7

So you may mention special versions of Axon Ivy (e.g. since 8.0 you need to ..)
but it looks good ..

File: src/app/pages/installation/_linux_designer.twig [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  1. Download the Axon Ivy Designer to your desired location
+<ul class="actions">
+  <li><a href="{{downloadUrl}}" class="button special"><i class="si si-download"></i> Download Axon Ivy Designer {{bugfixVersion}}</a></li>
+</ul>
+<img src="/images/installation/download-designer.png" />
+</p>
+
+<p>
+  2. Extract the downloaded zip file and navigate into the extract Axon.ivy Designer directory.
```
Comment (alexsuter) [inline src/app/pages/installation/_linux_designer.twig:10]: Axon Ivy Designer

Comment (alexsuter) [reply src/app/pages/installation/_linux_designer.twig:10]: extracted

File: src/app/pages/installation/_linux_designer.twig [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  <img src="/images/installation/linuxDesigner/ubuntuExtract.png" />
+</p>
+
+
+<p>
+  3. Run the <code>install-dependencies.sh</code> script in a Terminal, in order to install crucial dependencies.
```
Comment (alexsuter) [inline src/app/pages/installation/_linux_designer.twig:16]: terminal lowercase?

File: src/app/pages/installation/_linux_designer.twig [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  <img src="/images/installation/linuxDesigner/ubuntuRunInstallDeps.png" />
+  Supply your sudoer password in order to allow privileged installers to run:
+  <img src="/images/installation/linuxDesigner/ubuntuInstallDepsTerminal.png" />
+</p>
+
+<p>
+  4. If you work on Ubuntu 20.04 or newer, switch to X.org Window system by logging out, clicking on the settings button
```
Comment (alexsuter) [inline src/app/pages/installation/_linux_designer.twig:23]: why is this needed?

**Review by alexsuter — APPROVED** [review]

File: src/app/pages/installation/_linux_designer.twig (hunk 4 above)
Comment (ivy-rew) [reply src/app/pages/installation/_linux_designer.twig:23]: process-editor won't show any contents: maybe we can remove this once we have only the GLSP editor.
yet, there are also other minor issues on wayland, workspace selector is sometimes buggy.
ubuntu2110-awtBridge
ubuntuDesktop21 10 on QEMU-KVM_001

File: src/app/pages/installation/_linux_designer.twig (hunk 1 above)
Comment (ivy-rew) [reply src/app/pages/installation/_linux_designer.twig:1]: yet I think the guide fits for versions ... later I guess we will hide the Ubuntu section once we no longer depend upon X.org

File: src/app/pages/installation/_linux_designer.twig (hunk 4 above)
Comment (ivy-rew) [reply src/app/pages/installation/_linux_designer.twig:23]: Thanks for your fast feedback @alexsuter

**Comment by alexsuter** [comment]
ready2merge
